src/managers/GCommandLoader.js

```diff
@@ -48,23 +49,24 @@ class GCommandLoader {
      * @private
      */
     async __loadCommandFiles() {
-        for await (let file of (await fs.readdirSync(this.cmdDir))) {
-            const fileTypeIndex = (file.lastIndexOf('.') - 1 >>> 0) + 2;
-            const fileName = file.slice(0, fileTypeIndex - 1);
-            const fileType = file.slice(fileTypeIndex);
+        for await (let fsDirent of (await fs.readdirSync(this.cmdDir, { withFileTypes: true }))) {
+            let file = fsDirent.name;
+            // Note: fileName includes the extension (.)
+            const fileName = path.basename(file);
+            const fileType = path.extname(file);
```

Comment:
```suggestion
            const fileType = path.extname(file);
            const fileName = path.basename(file, fileType);
```
The file extension can be parsed into [`path.basename()`](https://nodejs.org/api/path.html#path_path_basename_path_ext) to remove it from the name, if that's desirable.